src/ast/operator.rs

```diff
@@ -1,3 +1,5 @@
+// SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
same copyright notice question here and in several other files of this PR below which I will not explicitly point out